src/reflect/scala/reflect/internal/tpe/GlbLubs.scala

```diff
@@ -501,11 +501,12 @@ private[internal] trait GlbLubs {
             val glbThisType = glbRefined.typeSymbol.thisType
             def glbsym(proto: Symbol): Symbol = {
               val prototp = glbThisType.memberInfo(proto)
-              val syms = for (t <- ts;
-                              alt <- (t.nonPrivateMember(proto.name).alternatives)
-                              if glbThisType.memberInfo(alt) matches prototp
-              ) yield alt
-              val symtypes = syms map glbThisType.memberInfo
+              val symtypes = for {
+                t   <- ts
+                alt <- t.nonPrivateMember(proto.name).alternatives
+                mi = glbThisType.memberInfo(alt)
+                if mi matches prototp
+              } yield mi
```

Comment:
Is this the same? I'm always suspicious of mid-stream assignment adding an extra map with tuple.
```
              ts.flatMap(_.nonPrivateMember(proto.name).alternatives.flatMap { alt =>
                val info = glbThisType.memberInfo(alt)
                if (info matches prototp) info :: Nil else Nil
              })
```

Reply:
I think it should be the same... and for comprehensions can be tricky. I can better use that code.